### PR TITLE
Make SqlServerTriggersTest less flaky by separating the triggers

### PR DESCRIPTION
**Pull request by AndriySvyryd**

### CODE REVIEW

File: test/Microsoft.EntityFrameworkCore.SqlServer.FunctionalTests/SqlServerTriggersTest.cs [hunk 1]
```diff
@@ -9,12 +9,13 @@
...
 namespace Microsoft.EntityFrameworkCore.SqlServer.FunctionalTests
 {
-    public class SqlServerTriggersTest : IClassFixture<SqlServerTriggersTest.SqlServerTriggersFixture>, IDisposable
+    public class SqlServerTriggersTest : IClassFixture<SqlServerTriggersTest.SqlServerTriggersFixture>
     {
         [Fact]
         public void Triggers_run_on_insert_update_and_delete()
         {
-            using (var context = CreateContext())
+            using (var testStore = GetTriggersTestStore())
```
Comment (anpete) [inline test/Microsoft.EntityFrameworkCore.SqlServer.FunctionalTests/SqlServerTriggersTest.cs]: Braces

**Comment by anpete** [comment]
:shipit:

File: test/Microsoft.EntityFrameworkCore.SqlServer.FunctionalTests/SqlServerTriggersTest.cs (hunk 1 above)
Comment (AndriySvyryd) [inline test/Microsoft.EntityFrameworkCore.SqlServer.FunctionalTests/SqlServerTriggersTest.cs]: I'm trying out the indent-less nested using pattern, looks better IMO

Comment (anpete) [inline test/Microsoft.EntityFrameworkCore.SqlServer.FunctionalTests/SqlServerTriggersTest.cs]: Would be great to discuss as a team

**Pull request merged**